src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -955,12 +954,16 @@ private async Task CalculatePRDescription(UpdateAssetsParameters update, List<De
                 coherencySection.AppendLine("attribute were produced in a build used as input to the parent dependency's build.");
                 coherencySection.AppendLine("See [Dependency Description Format](https://github.com/dotnet/arcade/blob/master/Documentation/DependencyDescriptionFormat.md#dependency-description-overview)");
                 coherencySection.AppendLine();
-
+                coherencySection.AppendLine("[DependencyUpdate]: <> (Begin)");
```

Comment:
The begin and end strings should be extracted as constants, as you are using them multiple times.